src/components/APOD/loading_spinner.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const LoadingSpinner: React.FC = () => (
+  <>
+    <div className="spinner-container">
+      <div className="loading-spinner"></div>
```

Comment:
can we center the loading spinner? `display: flex;` and `justify-content: center;` should do it

Reply:
Done!